model/clusters_mgmt/v1/cluster_resource.model

```diff
@@ -155,4 +155,8 @@ resource Cluster {
 	locator Hypershift {
 		target Hypershift
 	}
+
+	locator PrivateLinkConfiguration {
```

Comment:
If it's part of the aws struct, shouldn't it be under `.../clusters/id/aws/private_link_configuration`?

Reply:
but this one is to generate endpoint is /<clusterid>/private_link_configuration, so locator is under cluster.

Reply:
So you need another layer of `aws_resource` under cluster_resource, and place the locator there

Reply:
makes sense, done and updated design doc